Dockerfile.deps

```diff
@@ -0,0 +1,24 @@
+# Copyright 2021 Tetrate
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM gcr.io/cloud-builders/docker
+
+RUN apt-get -y update \
+ && apt-get -y install python3 python3-setuptools curl \
+ # `cloudbuild` images uses Ubuntu 16.04 as base which means Py 3.5
```

Comment:
I think this comment drifted.. was it copy/pasta?

```bash
docker history gcr.io/cloud-builders/docker --no-trunc
IMAGE                                                                     CREATED        CREATED BY                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       SIZE      COMMENT
sha256:41a8b353a145d653586327802f30bdfe088d4e8d3e264f50b5a8e225927bd8e1   16 hours ago   /bin/sh -c #(nop)  ENTRYPOINT ["/usr/bin/docker"]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                0B        
<missing>                                                                 16 hours ago   |1 DOCKER_VERSION=5:19.03.8~3-0~ubuntu-xenial /bin/sh -c apt-get -y update &&     apt-get -y install         apt-transport-https         ca-certificates         curl         make         software-properties-common &&     curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add - &&     apt-key fingerprint 0EBFCD88 &&     add-apt-repository        "deb [arch=amd64] https://download.docker.com/linux/ubuntu        xenial        edge" &&     apt-get -y update &&     apt-get -y install docker-ce=${DOCKER_VERSION} docker-ce-cli=${DOCKER_VERSION}   896MB     
<missing>                                                                 16 hours ago   /bin/sh -c #(nop)  ARG DOCKER_VERSION=5:19.03.8~3-0~ubuntu-xenial                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                0B        
<missing>                                                                 51 years ago   /tmp/installer.sh                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                0B        
<missing>                                                                 51 years ago   bazel build ...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  23.1MB    
<missing>                                                                 51 years ago   bazel build ...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  117MB     
```

Reply:
wdym? `xenial` is 16.04

Reply:
my eyes were crossed on this one. you are right!